src/rdiff_backup/eas_acls.py

```diff
@@ -101,9 +101,9 @@ def read_from_rp(self, rp):
     def clear_rp(self, rp):
         """Delete all the extended attributes in rpath"""
         try:
-            for name in rp.conn.xattr.listxattr(rp.path, rp.issym()):
+            for name in rp.conn.xattr.list(rp.path, rp.issym()):
                 try:
-                    rp.conn.xattr.removexattr(rp.path, name, rp.issym())
```

Comment:
hum, if we are doing this. We are breaking inter operability of the network protocole again ?

Reply:
Very valid remark, that's why I just created #416 and tested this MR against it, and didn't notice any issue.

This said, as xattr isn't an rdiff-backup module but an external one, calling it, even remotely, shouldn't make any difference from a network protocol perspective (only if e.g. `xattr.get` wouldn't exist on the other side, but with 12 years to go, we are on the very safe side).